spec/models/video_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Video, type: :model do
+  it "should be invalid with a slot of 5" do
+    video = build(:video, slot: 5)
+
+    expect(video).to be_invalid
+  end
+
+  it "should be invalid with a blank url" do
+    video = build(:video, url: "")
+    video.save
+
+    expect(video.errors.full_messages.to_sentence).to eql("Error Please enter a valid url.")
+    expect(video).to be_invalid
+  end
+
+  it "should ve valid with a lot of 1,2,3 or 4" do
+    video1 = build(:video, slot:1)
+    video2 = build(:video, slot:2)
+    video3 = build(:video, slot:3)
+    video4 = build(:video, slot:4)
```

Comment:
Maybe change one of these so it doesn't have a slot, just to show that it is still a valid video?